.github/workflows/matrix/ubuntu-x64-full-config.json

```diff
@@ -0,0 +1,85 @@
+{
+  "include": [
+    {
+      "os": "ubuntu20",
+      "config": "Debug",
+      "compiler": "Clang",
+      "cxx-compiler": "/usr/bin/clang++-11",
+      "c-compiler": "/usr/bin/clang-11",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu20",
+      "config": "Debug",
+      "compiler": "GCC",
+      "cxx-compiler": "/usr/bin/g++-10",
+      "c-compiler": "/usr/bin/gcc-10",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu20",
+      "config": "Release",
+      "compiler": "Clang",
+      "cxx-compiler": "/usr/bin/clang++-11",
+      "c-compiler": "/usr/bin/clang-11",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu20",
+      "config": "Release",
+      "compiler": "GCC",
+      "cxx-compiler": "/usr/bin/g++-10",
+      "c-compiler": "/usr/bin/gcc-10",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu22",
+      "config": "Debug",
+      "compiler": "Clang",
+      "cxx-compiler": "/usr/bin/clang++-14",
+      "c-compiler": "/usr/bin/clang-14",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu22",
+      "config": "Debug",
+      "compiler": "GCC",
+      "cxx-compiler": "/usr/bin/g++-12",
+      "c-compiler": "/usr/bin/gcc-12",
+      "cxx-standard": 23,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu22",
+      "config": "Release",
+      "compiler": "Clang",
+      "cxx-compiler": "/usr/bin/clang++-14",
+      "c-compiler": "/usr/bin/clang-14",
+      "cxx-standard": 20,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu22",
+      "config": "Release",
+      "compiler": "GCC",
+      "cxx-compiler": "/usr/bin/g++-12",
+      "c-compiler": "/usr/bin/gcc-12",
+      "cxx-standard": 23,
+      "build_mrcuda": "ON"
+    },
+    {
+      "os": "ubuntu24",
+      "config": "Debug",
+      "compiler": "Clang",
+      "cxx-compiler": "/usr/bin/clang++-18",
+      "c-compiler": "/usr/bin/clang-18",
+      "cxx-standard": 23,
+      "build_mrcuda": "ON"
+    }
```

Comment:
We have 4 different builds (Clang/GCC + Release/Debug) for Ubuntu 20 and 22, but only a single build for Ubuntu 24.

Reply:
For economy  we do not build Ubuntu24/GCC 13, while we have already Fedora/GCC 13